core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -187,6 +180,16 @@ func (eb *EthBroadcaster) Close() error {
 	})
 }
 
+// OptimisticClose closes the EthBroadcaster, returning nil if it was already
+// closed or not started yet
+func (eb *EthBroadcaster) OptimisticClose() error {
```

Comment:
nit/ maybe consider a different name to avoid confusion?
```suggestion
func (eb *EthBroadcaster) IdempotentClose() error {
```
Or would it make sense to have the common internal logic in a single shared helper so that callers can just pass the `io.Closer`?
```go
err := utils.CloseIdempotent(eb)
```

Reply:
`utils.EnsureClosed` :shrug: